server/room-battle.ts

```diff
@@ -730,12 +730,18 @@ export class RoomBattle extends RoomGames.RoomGame {
 		for (const player of this.players) player.wantsTie = false;
 
 		switch (lines[0]) {
-		case 'requesteddata':
+		case 'reseeded': {
```

Comment:
I guess this is only necessary to privately tell the host what the seed was. I think that's basically unnecessary anyway (the host can grab it from the inputlog after the battle is over if they really care), so my implementation doesn't use this.